clonelists/Nintendo - Nintendo DS (No-Intro).json

```diff
@@ -6215,6 +6247,24 @@
 				{"searchTerm": "Successfully Learning English"}
```

Comment:
Does this mean this also should be a compilation? Or that _Lernerfolg Grundschule - Englisch - Klasse 1-4_ should be added as a compilation of Year 2+3 and Year 4+5 instead?

Reply:
Thanks, will keep this in mind for next time.